bosh-stemcell/spec/stemcells/fips_spec.rb

```diff
@@ -39,7 +39,7 @@
     describe file('/boot/grub/grub.cfg') do
       it { should be_file }
       its(:content) { should match %r{linux\t/boot/vmlinuz-\S+-fips root=UUID=\S* ro } }
-      if ENV.key?("UBUNTU_FIPS_USE_IAAS_KERNEL")
+      if ENV.key?("UBUNTU_FIPS_USE_IAAS_KERNEL") and ENV["UBUNTU_FIPS_USE_IAAS_KERNEL"] != ""
```

Comment:
The `use_iaas_kernel` let from further down [below](https://github.com/cloudfoundry/bosh-linux-stemcell-builder/pull/309/commits/6582e8d911973925a0fd354f2fa5f44adc4cfe6a#diff-f18a6346d0ba4575fc99197220088b78b3e21a9696f0c5949166c0bb52463744R65) should be moved up so that it can be reused here.